components/fieldgroup/index.css

```diff
@@ -10,40 +10,78 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
+
+/* fieldgroup/index.css
+ *
+ * fieldgroup contains four component dependences:
+ * radio, checkbox, helptext, fieldlabel
+ *
+ */
+
+/* custom properties */
 .spectrum-FieldGroup {
-  --spectrum-fieldgroup-margin: var(--spectrum-global-dimension-size-200);
-  --spectrum-fieldgroup-readonly-delineator: '\002c';
+  --spectrum-fieldgroup-margin: var(--spectrum-spacing-300);
+  --spectrum-fieldgroup-readonly-delimiter: '\002c';
 }
 
+/* field group */
 .spectrum-FieldGroup {
   display: flex;
-  vertical-align: top;
+  align-items: top;
   flex-wrap: wrap;
+
+  /* field group label top aligned */
+  &--toplabel {
+    flex-direction: column;
+  }
+
+  /* field group label side aligned */
+  &--sidelabel {
+    flex-direction: row;
+  }
 }
 
+/* input fields layout */
+.spectrum-FieldGroupInputLayout {
+  display: flex;
+  align-items: top;
+  flex-wrap: wrap;
+
+  /* input fields stacked vertically */
+  &--vertical {
+    flex-direction: column;
+  }
+
+  /* input fields aligned horizontally */
+  &--horizontal {
+    flex-direction: row;
+
+    /* space between horizontal fields */
+    .spectrum-FieldGroup-item:not(:last-child) {
```

Comment:
nit, could we nest some of these definitions? 
It's also single line for a single rule so maybe, maybe not. Up to you, same below